### PR TITLE
Workflow that creates an issue with available updates

### PR DESCRIPTION
**Pull request by jmfayard**
Depends on https://github.com/krzema12/github-actions-kotlin-dsl/pull/170

### CODE REVIEW

File: wrapper-generator/src/main/kotlin/it/krzeminski/githubactions/wrappergenerator/versions/SuggestVersions.kt [hunk 1]
```diff
@@ -22,6 +23,7 @@ import it.krzeminski.githubactions.wrappergenerator.wrappersToGenerate
  * ```
  */
 fun main() {
+    var output: String = ""
```
Comment (krzema12) [inline wrapper-generator/src/main/kotlin/it/krzeminski/githubactions/wrappergenerator/versions/SuggestVersions.kt:26]: What if this variable remains empty? Will the workflow create the issue as well?

**Review by krzema12 — CHANGES_REQUESTED** [review]

File: .github/workflows/updates-available.main.kts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        uses(
+            name = "Run suggestVersions",
+            env = linkedMapOf("GITHUB_TOKEN" to expr("secrets.GITHUB_TOKEN")),
+            action = GradleBuildActionV2(
+                arguments = "suggestVersions",
+            )
+        )
+        uses(
+            name = "Create issue",
+            action = CustomAction(
```
[the remark this answers is not in the document]
Comment (jmfayard) [reply .github/workflows/updates-available.main.kts:47]: Yes but then we can't test the workflow until the library is released

File: wrapper-generator/src/main/kotlin/it/krzeminski/githubactions/wrappergenerator/versions/SuggestVersions.kt (hunk 1 above)
Comment (jmfayard) [reply wrapper-generator/src/main/kotlin/it/krzeminski/githubactions/wrappergenerator/versions/SuggestVersions.kt:26]: That would be great but currently not.
We could make `gradlew suggestVersions` fails, but then you receive an email so it's not better.
Ideally we would add a `condition = fileNotExists("suggestVersions.md)` but I'm afraid I don't know how to do that.

Comment (krzema12) [reply wrapper-generator/src/main/kotlin/it/krzeminski/githubactions/wrappergenerator/versions/SuggestVersions.kt:26]: Ok, let's keep it simple - we can resolve an empty ticket twice a month, worst case :)

File: .github/workflows/actions-versions.main.kts [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+import it.krzeminski.githubactions.domain.triggers.WorkflowDispatch
+import it.krzeminski.githubactions.dsl.expr
+import it.krzeminski.githubactions.dsl.workflow
+import java.nio.file.Paths
+
+val checkIfNewActionVersionsWorkflow = workflow(
+    name = "Updates available",
+    on = listOf(
+        Schedule(listOf(
+            Cron(dayMonth = "1,16", hour = "7", minute = "0")
```
Comment (krzema12) [inline .github/workflows/actions-versions.main.kts:19]: Let's make it once a week - that often I'm planning to release new versions, ideally.

Comment (jmfayard) [reply .github/workflows/actions-versions.main.kts]: Done, I've make it on fridays, you change change it to suit your needs

**Review by krzema12 — APPROVED** [review]

**Review by krzema12 — APPROVED** [review]

**Comment by jmfayard** [comment]
@krzema12 conflicts solved

**Review by krzema12 — APPROVED** [review]

**Pull request merged**